Gemfile

```diff
@@ -14,6 +14,7 @@ group :development, :test do
   gem "rails", '>= 3.1.0'
   gem "rspec", "~> 2.10.0"
   gem "awesome_print"
+  gem "byebug"
```

Comment:
Adding byebug seems to have broken the Ruby 1.9.3 build. Can you look into fixing that?